lib/helpers/DataTypeUiHelper.ts

```diff
@@ -999,4 +1012,47 @@ export class DataTypeUiHelper extends UiBaseLocators {
   async getAddButtonInGroupWithName(name: string) {
     return this.page.locator('.group').filter({hasText: name}).locator('#add-button');
   }
+
+  // Tiptap
+  async deleteToolbarGroup(groupIndex: number, rowIndex: number = 0) {
+    const groupButton = this.tiptapToolbarConfiguration.locator('.row').nth(rowIndex).locator('.group').nth(groupIndex);
+    await groupButton.hover();
+    await groupButton.locator('[label="Remove group"]').click();
+  }
+
+  async deleteToolbarRow(rowIndex: number) {
+    const rowButton = this.tiptapToolbarConfiguration.locator('.row').nth(rowIndex);
+    await rowButton.hover();
+    await rowButton.locator('[label="Remove group"]').click();
+  }
+
+  async clickAddRowToolbarButton() {
+    await this.addRowToolbarBtn.click();
+  }
+
+  async clickAddGroupToolbarButton() {
+    await this.addGroupToolbarBtn.click();
+  }
+
+  async clickExtensionItemHaveName(name: string) {
+    await this.tiptapExtensionsConfiguration.locator('uui-checkbox[label="' + name + '"]').click();
+  }
+
+  async addToolbarItemToGroup(itemName: string, rowIndex: number = 0) {
```

Comment:
Should we specify the group Index for this?

Reply:
Removed it as it is unnecessary now.